clang/docs/ReleaseNotes.rst

```diff
@@ -117,6 +117,11 @@ C23 Feature Support
 Non-comprehensive list of changes in this release
 -------------------------------------------------
 
+* Code compiled with ``-shared`` and ``-ffast-math`` will no longer enable
+  flush-to-zero floating-point mode by default. This decision can be overridden
```

Comment:
You may want to revise this to clarify that this only refers to the use of -shared and -ffast-math on the command-line used to link the program.